### ISSUE TITLE
[Bug]: Alt text is not being displayed when image is not available

### ISSUE THREAD

**Issue by rennie-moodley** [markview-app/support#40]
### Description

Hi Team

When adding images to the markdown files `![logo](./docs/assets/images/contributing-logo.png),` it renders fine when the image is available, however when the image fails to load, the "alt text" is not being displayed.

The preview does not indicate that an image is missing.

### Steps to Reproduce

1. Create a markdown file with the following `![logo](./docs/assets/images/contributing.png)`
2. Observe that the markdown does not render the alt text

### MarkView Version

v2.8.2

### Browser

Chrome

### Operating System

Windows

### Before submitting

- [x] I have searched existing issues to avoid duplicates
- [x] I have removed any sensitive information from screenshots/logs

**Comment by thea-vu** [comment]
Thanks for reporting this, @rennie-moodley! 

It's fixed and will ship in `v2.8.6`.

**Comment by rennie-moodley** [comment]
Thanks @thea-vu 

I have tested and confirmed that this is fixed in 2.8.6.
